### PR TITLE
Create salesforce-aura-objects.txt

### PR DESCRIPTION
**Pull request by tacticthreat**
Salesforce Aura Standard Objects

### CODE REVIEW

**Comment by ItsIgnacioPortal** [comment]
How did you make this list?

**Comment by tacticthreat** [comment]
Compiled and cleaned it up from Salesforces' documentation, also used it in a webinar collab with Bugcrowd relating to Salesforce Object Recon and exploiting "/s/sfsites/aura" endpoint.

**Review by ItsIgnacioPortal — APPROVED** [review]
thanks for contributing!

**Review by ItsIgnacioPortal — CHANGES_REQUESTED** [review]
Oh right, you still need to remove the comment at the beginning of the file

**Review by tacticthreat — COMMENTED** [review]
Could be helpful reminder but I understand

File: Discovery/Web-Content/api/salesforce-aura-objects.txt [hunk 1]
```diff
@@ -0,0 +1,1075 @@
...
+EmailServicesFunction
+EmailStatus
+EmailTemplate
+EmailTemplateMonthlyMetric
+EmbeddedServiceDetail
+EmbeddedServiceLabel
+Employee
+EmployeeCrisisAssessment
+Emp
+Emp
```
Comment (ItsIgnacioPortal) [inline Discovery/Web-Content/api/salesforce-aura-objects.txt:375]: Make sure to run `uniq` on this file to remove duplicates :p

**Review by ItsIgnacioPortal — CHANGES_REQUESTED** [review]

File: Discovery/Web-Content/api/salesforce-aura-objects.txt (hunk 1 above)
Comment (tacticthreat) [reply Discovery/Web-Content/api/salesforce-aura-objects.txt:375]: Removed duplicate values

**Comment by g0tmi1k** [comment]
Thanks @ItsIgnacioPortal

**Pull request merged**